lib/src/gateway/gateway.dart

```diff
@@ -1162,6 +1184,37 @@ class Gateway extends GatewayManager with EventParser {
     }
   }
 
+  SoundboardSoundCreateEvent parseSoundboardSoundCreate(Map<String, Object?> raw) {
+    final guildId = maybeParse(raw['guild_id'], Snowflake.parse);
+
+    return SoundboardSoundCreateEvent(
+      gateway: this,
+      sound: client.guilds[guildId ?? Snowflake.zero].soundboard.parse(raw),
```

Comment:
This shouldn't default to the zero guild's soundboard if guildId is null, it should default to client.sounboard.

Same for other instances of accessing things with `?? Snowflake.zero`.

Reply:
Alright, fair enough. Didn't realise these events couldn't be triggered by global events.